### PR TITLE
Add labels

### LINKED ISSUE #122: Update UI
Add labels for client window on left hand side, and meeting window on right hand side

### PR DESCRIPTION
**Pull request by itstrueitstrueitsrealitsreal**
- Added labels for client and meetings
- Fixes #122

### CODE REVIEW

File: src/main/resources/view/PersonListPanel.fxml [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
+<?import javafx.scene.layout.*?>
+<?import javafx.scene.shape.*?>
 
-<VBox xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
+<VBox xmlns="http://javafx.com/javafx/17.0.2-ea" xmlns:fx="http://javafx.com/fxml/1">
+  <VBox alignment="CENTER_LEFT" spacing="5">
+    <padding>
+      <Insets bottom="5" />
+    </padding>
+    <Label fx:id="clientList" styleClass="title_label" text="Clients" textFill="WHITE" />
```
Comment (timothysashimi) [inline src/main/resources/view/PersonListPanel.fxml:13]: Good to add client list to inform users that the list contains clients

File: src/main/resources/view/MeetingListPanel.fxml [hunk 2]
```diff
@@ -1,8 +1,18 @@
...
+<?import javafx.scene.layout.*?>
+<?import javafx.scene.shape.*?>
 
-<VBox xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
+<VBox xmlns="http://javafx.com/javafx/17.0.2-ea" xmlns:fx="http://javafx.com/fxml/1">
+    <VBox alignment="CENTER_LEFT" spacing="5">
+        <padding>
+            <Insets bottom="5" />
+        </padding>
+        <Label fx:id="meetingList" styleClass="title_label" text="Meetings" textFill="#eeeeee" />
```
Comment (timothysashimi) [inline src/main/resources/view/MeetingListPanel.fxml:13]: Good to add client list to inform users that the list contains clients

File: src/main/resources/view/MeetingListPanel.fxml [hunk 3]
```diff
@@ -1,8 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<?import javafx.scene.control.ListView?>
-<?import javafx.scene.layout.VBox?>
+<?import javafx.geometry.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+<?import javafx.scene.shape.*?>
```
Comment (jaejayrome) [inline src/main/resources/view/MeetingListPanel.fxml:6]: Good ordering of your import statements, following the checkstyle convention!

File: src/main/resources/view/PersonListPanel.fxml [hunk 4]
```diff
@@ -1,8 +1,18 @@
...
+    <Line endX="125.0" endY="0.0" startX="0.0" startY="0.0" stroke="WHITE" strokeType="OUTSIDE" styleClass="separation_line">
+    </Line>
+  </VBox>
```
Comment (jaejayrome) [inline src/main/resources/view/PersonListPanel.fxml:16]: I like the choice of colour of your line, matches the background colour of FinCliq!

File: src/main/resources/view/DarkTheme.css [hunk 5]
```diff
@@ -17,6 +17,11 @@
     -fx-opacity: 1;
 }
 
+.title_label {
+    -fx-font-size: 20px;
+    -fx-text-fill: #FFFFFF;
```
Comment (jaejayrome) [inline src/main/resources/view/DarkTheme.css:22]: Perhaps you can consider using rgb values for your colour rather than hexadecimal numbering

**Review by jaejayrome — COMMENTED** [review]
LGTM!

File: src/main/resources/view/DarkTheme.css [hunk 6]
```diff
@@ -332,6 +337,10 @@
     -fx-effect: innershadow(gaussian, black, 10, 0, 0, 0);
 }
 
+#filterField, #meetingListPanel, #personWebpage {
+    -fx-effect: innershadow(gaussian, black, 10, 0, 0, 0);
+}
+
```
Comment (sethteo) [inline src/main/resources/view/DarkTheme.css:343]: The effect looks good!

File: src/main/resources/view/PersonListPanel.fxml (hunk 1 above)
Comment (sethteo) [inline src/main/resources/view/PersonListPanel.fxml:13]: Very clean looking UI!

File: src/main/resources/view/DarkTheme.css [hunk 7]
```diff
@@ -17,6 +17,11 @@
     -fx-opacity: 1;
 }
 
+.title_label {
```
Comment (chewbum) [inline src/main/resources/view/DarkTheme.css:20]: Good design!

File: src/main/resources/view/DarkTheme.css [hunk 8]
```diff
@@ -332,6 +337,10 @@
     -fx-effect: innershadow(gaussian, black, 10, 0, 0, 0);
 }
 
+#filterField, #meetingListPanel, #personWebpage {
```
Comment (chewbum) [inline src/main/resources/view/DarkTheme.css:340]: good job kenneth!

File: src/main/resources/view/MeetingListPanel.fxml [hunk 9]
```diff
@@ -1,8 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<?import javafx.scene.control.ListView?>
-<?import javafx.scene.layout.VBox?>
+<?import javafx.geometry.*?>
+<?import javafx.scene.control.*?>
```
Comment (chewbum) [inline src/main/resources/view/MeetingListPanel.fxml:4]: Thanks for following the coding standards!

**Pull request merged**